hw04_lru_cache/go.mod

```
@@ -1,4 +1,4 @@
-module github.com/fixme_my_friend/hw04_lru_cache
+module github.com/N1shko/hw04_lru_cache
```

Нужен полный путь к пакету, включая название репозитория:
```suggestion
module github.com/N1shko/otus-golang/hw04_lru_cache
```